src/dialogs.ycp

```diff
@@ -77,20 +77,20 @@ term EditDialogContents()
 					    `VBox(
 						  // combo box label: enable FCoE (yes/no)
 						  `ComboBox( `id(`fcoe), _("&FCoE Enable"),
-							     [`item(`id("yes"), "yes" ),
-							      `item(`id("no"), "no", true ) ]
+							     [`item(`id("yes"), _("Yes") ),
```

Comment:
I have checked the [Label](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Label.rb) and the [Message](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Message.rb) and surprisingly we do not have such common texts defined. `Label.Yes/NoButton` contains the `&` keyboard shortcut marker and cannot be used here.

But I found [`Yast::String.YesNo`](https://github.com/yast/yast-yast2/blob/562e2f3ccfa8413a0528cf067ea7533fa72a2450/library/types/src/modules/String.rb#L99).

I'd suggest something like this:

```ruby
def yes_str
  Yast::String.YesNo(true)
end

def no_str
  Yast::String.YesNo(false)
end
...
[`item(`id("yes"), yest_str, ...]
```

Or you could even DRY it a bit more:

```ruby
def yesno_list(default)
  [`item(`id("yes"), yes_str, default == :yes ), `item(`id("no"), no_str, default == :no )]
end
...
ComboBox( `id(`fcoe), _("&FCoE Enable"), yesno_list(:no))
```